lfucg-exactions/client/js/components/AgreementForm.js

```diff
@@ -120,10 +120,17 @@ class AgreementForm extends React.Component {
                                         </div>
                                     </div>
                                     <div className="row">
-                                        <div className="col-sm-6">
-                                            <FormGroup label="Agreement Type" id="agreement_type">
-                                                <input type="text" className="form-control" placeholder="Agreement Type" />
-                                            </FormGroup>
+                                        <div className="col-sm-6 form-group">
+                                            <label htmlFor="agreement_type" className="form-label" id="agreement_type" aria-label="Agreement Type">Agreement Type</label>
+                                            <select className="form-control" id="agreement_type" onChange={formChange('agreement_type')} >
+                                                {agreements.agreement_type ? (
+                                                    <option value="agreement_type" aria-label={`Agreement Type ${agreements.agreement_type_display}`}>{agreements.agreement_type_display}</option>
```

Comment:
Adding display on the end presents the display option from a tuple.  The different kinds of quotes allows for incorporation of data.